node/service/lib/device.d.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+declare class Device {
+    deviceId: string;
+    generationId: string;
+    etag: string;
+    connectionState: string;
+    status: 'enabled' | 'disabled';
```

Comment:
should double-check with the service team that these are the only two possibilities